### ISSUE TITLE
[Enhancement Request] Upload kodi.old.log to paste.kodi.tv from FEN

### ISSUE THREAD

**Issue by TwilightMercy** [Tikipeter/repository.tikipeter#218]
Hi,

in **FEN --> Log Utils**

![image](https://user-images.githubusercontent.com/29895680/234319290-90a94586-9761-4679-8dce-070d6643c581.png)

the Upload Kodi Log to Pastebin (paste.kodi.tv actually) only uploads the latest kodi.log file.
Is it possible to upload also kodi.old.log file? after Kodi crashed, uploading the regular kodi.log file doesn't show up any useful information. and it is useful for debugging issues.

Thanks!

**Comment by Tikipeter** [comment]
Done for next update.

**Comment by TwilightMercy** [comment]
Thanks

**Comment by TwilightMercy** [comment]
Hi Tiki,

Thank you very much for adding the Old log file upload feature!
I've downloaded from test repostiry FEN version 3.1.08b, and it works great:

![image](https://user-images.githubusercontent.com/29895680/235296695-f95390d1-0dd5-41a5-b02b-a7bcdfecfd05.png)


However, I have another small request if possible (for you decision) -
In my build I'm running (for my big Kodi telegram group), I've added a shortcut in Kodi favorites to upload log from FEN. Example: (Ignore the Hebrew text for rest of FEN buttons)

![image](https://user-images.githubusercontent.com/29895680/235296850-a2a91682-a90e-41ea-9a1b-1b36ae0897ec.png)


This button is direct shortcut to "Are you sure" confirm dialog, which uploads kodi.log file.
In order to avoid two different buttons , one for kodi.log and one for kodi.old.log, Example:
![image](https://user-images.githubusercontent.com/29895680/235297020-c03f8289-8c6c-467d-a2d3-109010d077b6.png)


Is it possible to change the menu to only "Upload Kodi Log to Pastebin" - and clicking on it will load a dialog to choose between kodi.log and kodi.old.log?
So it will be under one mutual button : )
